mep/books/migrations/0021_creator_type_order.py

```diff
@@ -0,0 +1,51 @@
+# Generated by Django 2.2.11 on 2020-03-31 19:41
+
+from django.db import migrations, models
+
+# default ordering for some common creator types
+CREATOR_TYPE_ORDER = {
+    'Author': 1,
+    'Editor': 2,
+    'Illustrator': 3,
+    'Introduction': 4,
+    'Preface': 5,
+    'Translator': 6,
+}
+
+def set_default_creator_type_ordering(apps, _schema_editor):
+    '''Set sensible order values for some pre-existing creator types.'''
+
+    CreatorType = apps.get_model('books', 'CreatorType')
+    for creator_type in CreatorType.objects.all():
+        if creator_type.name in CREATOR_TYPE_ORDER:
```

Comment:
Could have used dict `get` method with a default value here so you don't have to check if they're in the list (but not worth refactoring).